### PR TITLE
ci: cleanup up test app directories

### PR DESCRIPTION
**Pull request by jkleinsc**
#### Description of Change

This PR cleans up directories on WOA that tests create so that the WOA runners are in a clean state for subsequent test runs. 
#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).
- [x] This is **NOT A BREAKING CHANGE**. Breaking changes may not be merged to master until 11-x-y is branched.

#### Release Notes

Notes: none

### CODE REVIEW

File: azure-pipelines-woa.yml [hunk 1]
```diff
@@ -95,3 +95,19 @@ steps:
...
+    Remove-Item -path $env:APPDATA/crash -Recurse
+    Remove-Item -path $env:APPDATA/default-menu -Recurse
+    Remove-Item -path $env:APPDATA/document-idle -Recurse
+    Remove-Item -path $env:APPDATA/foo -Recurse
+    Remove-Item -path $env:APPDATA/initial-app -Recurse
+    Remove-Item -path $env:APPDATA/locale-check -Recurse
+    Remove-Item -path $env:APPDATA/snapshot-items-available -Recurse    
+    Remove-Item -path $env:APPDATA/site-instance-overrides -Recurse
+    Remove-Item -path $env:APPDATA/remote-control -Recurse
+    Remove-Item -path $env:APPDATA/window-all-closed -Recurse
```
Comment (nornagon) [inline azure-pipelines-woa.yml:111]: oof. new tests that do this stuff aren't going to remember to add to this list.

can we change the tests to all put their data in a specific top-level dir and then delete that? eg. %APPDATA%\electron-tests\[stuff]?

Most tests that do this sort of thing do it by copy-pasting another test, so it'd likely stay good.

Comment (jkleinsc) [reply azure-pipelines-woa.yml:111]: sure.  we could do that

**Comment by jkleinsc** [comment]
@nornagon we already are deleting electron* directories:
https://github.com/electron/electron/blob/master/azure-pipelines-woa.yml#L95

**Pull request merged**